prompting/zero-shot.py

```diff
@@ -28,4 +28,13 @@ def generate_response(self, prompt, model="gemini-2.0-flash"):
     print(f"Response: {response}")
     
     # Output:
-    # Response: The capital of France is Paris.
\ No newline at end of file
+    # Response: The capital of France is Paris.
+
+
+def generatePassword(length=12):
+    import random
+    import string
+
+    characters = string.ascii_letters + string.digits + string.punctuation
```

Comment:
Consider adding error handling to handle exceptions when generating a password, such as KeyError or IndexError.